src/app/content/components/SearchResultsSidebar.tsx

```diff
@@ -0,0 +1,278 @@
+import { SearchResult } from '@openstax/open-search-client/dist/models/SearchResult';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+import styled, { css } from 'styled-components/macro';
+import { Search } from 'styled-icons/fa-solid/Search';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import { Details, iconSize } from '../../components/Details';
+import { navDesktopHeight, navMobileHeight } from '../../components/NavBar';
+import { labelStyle, textRegularLineHeight, textRegularStyle } from '../../components/Typography';
+import theme from '../../theme';
+import { AppState } from '../../types';
+import * as selectSearch from '../search/selectors';
+import {
+  bookBannerDesktopMiniHeight,
+  bookBannerMobileMiniHeight,
+  mobileSearchContainerMargin,
+  searchResultsBarDesktopWidth,
+  searchResultsBarMobileWidth,
+  sidebarDesktopWidth,
+  sidebarMobileWidth,
+  toolbarDesktopHeight,
+  toolbarMobileHeight,
+  toolbarSearchInputMobileHeight,
+} from './constants';
+import { CollapseIcon, ExpandIcon, Summary, SummaryTitle, SummaryWrapper } from './Sidebar/styled';
+import { toolbarIconStyles } from './Toolbar';
+
+const searchResultsBarVariables = {
+    backgroundColor: '#f1f1f1',
+    mainPaddingDesktop: 3,
+    mainPaddingMobile: 2,
+};
+
+// tslint:disable-next-line:variable-name
+const SearchIconInsideBar = styled(Search)`
+  ${toolbarIconStyles}
+  color: ${theme.color.primary.gray.darker};
+  margin-right: 0.7rem;
+  margin-left: 3rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const CloseIcon = styled(Times)`
+  ${toolbarIconStyles}
+  color: ${theme.color.primary.gray.lighter};
+  margin-right: 1.4rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const SearchResultsBar = styled.div`
```

Comment:
most of this is the same as the ToC wrapper, we should try to combine as much as possible